worker-build/src/js/shim.js

```diff
@@ -4,4 +4,4 @@ export * from "./index_bg.js";
 // Run the worker's initialization function.
 imports.start?.();
 
-export default { fetch: imports.fetch, scheduled: imports.scheduled, queue: imports.queue };
+export default { connect: imports.connect, fetch: imports.fetch, scheduled: imports.scheduled, queue: imports.queue };
```

Comment:
These are the exported event handlers, I'm not *aware* of any inbound TCP handlers yet so unless this is a sneak peek into inbound socket workers (👀), it should be fine to remove the addition haha

Reply:
No, I wasn't sure how the global scope was constructed, I will remove it.